services/skus/receipt.go

```diff
@@ -14,53 +14,24 @@ import (
 	"time"
 
 	"github.com/awa/go-iap/appstore"
-	"github.com/awa/go-iap/playstore"
-	"google.golang.org/api/androidpublisher/v3"
-
 	appctx "github.com/brave-intl/bat-go/libs/context"
 	"github.com/brave-intl/bat-go/libs/logging"
+	"google.golang.org/api/androidpublisher/v3"
+	"google.golang.org/api/option"
 
 	"github.com/brave-intl/bat-go/services/skus/model"
 )
 
 const (
-	androidPaymentStatePending int64 = iota
-	androidPaymentStatePaid
-	androidPaymentStateTrial
-	androidPaymentStatePendingDeferred
-)
-
-const (
-	androidCancelReasonUser      int64 = 0
-	androidCancelReasonSystem    int64 = 1
-	androidCancelReasonReplaced  int64 = 2
-	androidCancelReasonDeveloper int64 = 3
-
-	purchasePendingErrCode       = "purchase_pending"
-	purchaseDeferredErrCode      = "purchase_deferred"
-	purchaseStatusUnknownErrCode = "purchase_status_unknown"
-	purchaseFailedErrCode        = "purchase_failed"
-	purchaseValidationErrCode    = "validation_failed"
+	purchasePendingErrCode    = "purchase_pending"
+	purchaseExpiredErrCode    = "purchase_expired"
+	purchaseValidationErrCode = "validation_failed"
```

Comment:
note: this is shared by both ios and android, we may want to update this as its not very descriptive and maybe confusing.

Reply:
I'd opt for doing it as a follow-up.